src/components/PostCared.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import service from "../appwrite/conf";
+import { Link } from "react-router-dom";
+
+const PostCared = ({ $id, title, featuredImage }) => {
```

Comment:
what is this component about?

Reply:
It's about showing the preview of blogs when we fetch them ...
And every time you click on a card it will open that blog on the next page